pkg/genevalogging/genevalogging.go

```diff
@@ -32,6 +32,12 @@ const (
 	mdsdImage      = "arosvc.azurecr.io/genevamdsd:master_249"
 
 	parsersConf = `
+[PARSER]
+	Name audit
+	Format json
+	Time_Key stageTimestamp
+	Time_Format %Y-%m-%dT%H:%M:%S.%L
```

Comment:
does %L truncate microseconds?  I think this is a copy/paste from the crio parser and I think that the %L use there is wrong too.  Should both be %N?

Reply:
Fluentbit implements its own factional seconds parsing in
https://github.com/fluent/fluent-bit/blob/master/src/flb_parser.c#L223
https://github.com/fluent/fluent-bit/blob/master/src/flb_parser.c#L734
https://github.com/fluent/fluent-bit/blob/master/src/flb_parser.c#L773

and supports only %L for fractions of a second (and not as a real input field descriptor - can only be used in conjunction with %s or %S). It doesn't have the %N found in Ruby parsers, but uses strtod to convert a decimal notation (starting with ".") to a double, so it parses fractions of a second no matter the number of digits used.